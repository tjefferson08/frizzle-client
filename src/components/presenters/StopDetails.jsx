import React from 'react';

const StopDetails = ({ params }) => {
  return (
    <div>
      <h3>{params.stop_id}</h3>
      <div>
        I'm a details card.
      </div>
    </div>
  );
};

export default StopDetails;
