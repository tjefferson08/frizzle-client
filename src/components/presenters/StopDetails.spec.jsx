import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
import StopDetails from './StopDetails';

test('should render a UL of stops', () => {
  const renderer = ReactTestUtils.createRenderer();
  const fakeStops = [{ stop_name: 'Stop 1' }, { stop_name: 'Stop 2' }];
  renderer.render(<StopDetails stop_id="123" />);
  const result = renderer.getRenderOutput();

  expect(result.type).toBe('div');
  expect(result).toMatchSnapshot();
});

