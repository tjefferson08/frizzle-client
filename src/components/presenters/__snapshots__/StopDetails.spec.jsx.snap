exports[`test should render a UL of stops 1`] = `
<div>
  <h3>
    123
  </h3>
  <div>
    I\'m a details card.
  </div>
</div>
`;
